pkgs/development/python-modules/pydyf/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib,
+  buildPythonPackage,
+  fetchPypi,
+  isPy3k,
+  pytestCheckHook,
+  coverage,
+  ghostscript,
+  pillow,
+  pytest,
+  pytest-cov,
+  pytest-flake8,
+  pytest-isort
+}:
+
+buildPythonPackage rec {
+  pname = "pydyf";
+  version = "0.1.2";
+  disabled = !isPy3k;
+
+  pytestFlagsArray = [
+    # setup.py is auto-generated and doesn't pass the flake8 check
+    "--ignore=setup.py"
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    coverage
+    ghostscript
+    pillow
+    pytest
+    pytest-cov
+    pytest-flake8
+    pytest-isort
```

Comment:
pytest should not be added to checkInputs when using pytestCheckHook. in pythonPackages we never want to generate cov reports because it just takes time and there is no advantage in doing it. flake8 tests are not run because they easily break on upgrades and don't provide any benefit. Doing a PR with those changes right now.

Reply:
Makes sense, thanks!